`gh gei reclaim-mannequin`'s single user (not CSV) flow does not work

## Description

The `gh gei reclaim-mannequin`'s documented option to specify a `--mannequin-user` and `--target-user` does not work.

Even when both arguments are specified, you get the following error:

```
[11:50] [ERROR] Either --csv or --mannequin-user and --target-user must be specified
```

Specifying `--csv` works as expected.

## Reproduction Steps

Run `gh gei reclaim-mannequin --github-target-org Deep-Avocado-9 --mannequin-user foo --target-user foo` on `gh-gei` v0.31.

--- a/RELEASENOTES.md
+++ b/RELEASENOTES.md
@@ -1,2 +1,3 @@
 - Added logic to check if a target repo exists before generating GHES archives
+- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were were not being passed to the reclaim command
 

--- a/RELEASENOTES.md
+++ b/RELEASENOTES.md
@@ -1,3 +1,3 @@
 - Added logic to check if a target repo exists before generating GHES archives
-- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were were not being passed to the reclaim command
+- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command
 - Added logic to ensure target org exists before generating GHES archives

--- a/RELEASENOTES.md
+++ b/RELEASENOTES.md
@@ -1,3 +1,3 @@
 - Added logic to check if a target repo exists before generating GHES archives
-- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command
+- Fixed reclaiming a single mannequin using `reclaim-mannequin` with the `--mannequin-user` and `--target-user` parameters
 - Added logic to ensure target org exists before generating GHES archives

--- a/src/Octoshift/Commands/ReclaimMannequinCommandBase.cs
+++ b/src/Octoshift/Commands/ReclaimMannequinCommandBase.cs
@@ -33,7 +33,7 @@ public class ReclaimMannequinCommandBase : CommandBase<ReclaimMannequinCommandAr
         Description = "CSV file path with list of mannequins to be reclaimed."
     };
 
-    public virtual Option<string> MannequinUsername { get; } = new("--mannequin-user")
+    public virtual Option<string> MannequinUser { get; } = new("--mannequin-user")
     {
         Description = "The login of the mannequin to be remapped."
     };
@@ -43,7 +43,7 @@ public class ReclaimMannequinCommandBase : CommandBase<ReclaimMannequinCommandAr
         Description = "The Id of the mannequin, in case there are multiple mannequins with the same login you can specify the id to reclaim one of the mannequins."
     };
 
-    public virtual Option<string> TargetUsername { get; } = new("--target-user")
+    public virtual Option<string> TargetUser { get; } = new("--target-user")
     {
         Description = "The login of the target user to be mapped."
     };
@@ -84,9 +84,9 @@ public class ReclaimMannequinCommandBase : CommandBase<ReclaimMannequinCommandAr
     {
         AddOption(GithubOrg);
         AddOption(Csv);
-        AddOption(MannequinUsername);
+        AddOption(MannequinUser);
         AddOption(MannequinId);
-        AddOption(TargetUsername);
+        AddOption(TargetUser);
         AddOption(Force);
         AddOption(GithubPat);
         AddOption(Verbose);

